### ISSUE TITLE
Problem with After update from 5.4.10 to 6.2.0

### ISSUE THREAD

**Issue by logrisua** [RocketChat/Rocket.Chat#29388]
### Description:

After update Rocket.Chat server to version 6.2.0 SAML login stop woking when 2FA enabled. If I disable 2FA login work correct.

### Steps to reproduce:

Udate Rocket.Chat server from 5.4.10 to 6.2.0

### Server Setup Information:

- Version of Rocket.Chat Server: 6.2.0
- Operating System: Rocky8
- Deployment Method: docker
- Number of Running Instances: 
- DB Replicaset Oplog: 
- NodeJS Version: v14.21.3
- MongoDB Version: 5.0.18

### Client Setup Information

- Desktop App or Browser Version: Decktop App v3.9.4 or any Browser
- Operating System: Windows

**Comment by hugocostadev** [comment]
Are you still experiencing this issue? Thanks

**Comment by logrisua** [comment]
Yes. I could not find a solution to this problem. Except return to version 5.1.8, where everything worked as it should.

**Comment by arthur-trt** [comment]
Had the same problem, we had to disable TOTP for everyone.
